### PR TITLE
Add Bluetooth source_type icon

### PR DESCRIPTION
**Pull request by Mariusthvdb**
and extend default icon to differentiate between home/not_home

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/common/entity/domain_icon.ts [hunk 1]
```diff
@@ -51,7 +54,10 @@ export const domainIcon = (
       if (stateObj?.attributes.source_type === "router") {
         return compareState === "home" ? mdiLanConnect : mdiLanDisconnect;
       }
-      return mdiAccount;
+      if (stateObj?.attributes.source_type === "bluetooth") {
+        return compareState === "home" ? mdiBluetooth : mdiBluetoothOff;
+      }
+      return compareState === "home" ? mdiAccount : mdiAccountOff;
```
Comment (bramkragten) [inline src/common/entity/domain_icon.ts:60]: A device tracker can also be in a zone, so this is not correct

File: src/common/entity/domain_icon.ts [hunk 2]
```diff
@@ -51,7 +54,10 @@ export const domainIcon = (
       if (stateObj?.attributes.source_type === "router") {
         return compareState === "home" ? mdiLanConnect : mdiLanDisconnect;
       }
-      return mdiAccount;
+      if (stateObj?.attributes.source_type === "bluetooth") {
```
Comment (bramkragten) [inline src/common/entity/domain_icon.ts:57]: add `bluetooth_le`

File: src/common/entity/domain_icon.ts [hunk 3]
```diff
@@ -51,7 +54,10 @@ export const domainIcon = (
...
+        return compareState === "home" ? mdiBluetooth : mdiBluetoothOff;
```
Comment (bramkragten) [inline src/common/entity/domain_icon.ts:58]: Don't like the `off`... Maybe
```suggestion
        return compareState === "home" ? mdiBluetoothConnect : mdiBluetooth;
```

File: src/common/entity/domain_icon.ts (hunk 1 above)
Comment (bramkragten) [reply src/common/entity/domain_icon.ts:60]:
```suggestion
      return compareState === "not_home" ? mdiAccountArrowRight : mdiAccount;
```

File: src/common/entity/domain_icon.ts (hunk 2 above)
Comment (Mariusthvdb) [reply src/common/entity/domain_icon.ts:57]: can we write 
```
if (stateObj?.attributes.source_type in ["bluetooth","bluetooth_le"] )
```
?

Comment (zsarnett) [reply src/common/entity/domain_icon.ts]: Thats SQL haha

Javascipt"

```js
if ( ["bluetooth","bluetooth_le"].includes(stateObj?.attributes.source_type))
```

Comment (Mariusthvdb) [reply src/common/entity/domain_icon.ts]: thx....

**Comment by Mariusthvdb** [comment]
must be late, I dont get it. No prettier warnings in vscode editor.

**Comment by Mariusthvdb** [comment]
please lend me a hand, I am lost for now why this wont check the lint/prettier

**Comment by Mariusthvdb** [comment]
thanks to Ludeeus for posting me to https://prettier.io/ .... online github.dev vscode editor doesn't really check lint issues apparently.

**Comment by bramkragten** [comment]
Please remove `.vscode/settings.json`

**Comment by Mariusthvdb** [comment]
no idea how that got there... my pardon

**Review by bramkragten — APPROVED** [review]

**Pull request merged**